server/server.go

```diff
@@ -49,6 +49,7 @@ const (
 	certRefreshInterval            = time.Minute * 5
 	rootlessEnvName                = "_CRIO_ROOTLESS"
 	irqBalanceConfigRestoreDisable = "disable"
+	eventStaggerDuration           = 50 * time.Millisecond
```

Comment:
have you tested this with Spegel yet? is this stagger enough?

Reply:
I need to check this data once.

Reply:
I'm not using this stuff anymore. Leftover from the previous push.

Reply:
I guess for environments with relatively low event frequency, a stagger duration of around 100-200 milliseconds may suffice.